qiskit_nature/second_q/mappers/bosonic_mapper.py

```diff
@@ -0,0 +1,54 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Bosonic Mapper."""
+
+from __future__ import annotations
+
+from qiskit.opflow import PauliSumOp
+from qiskit.quantum_info import SparsePauliOp
+from qiskit_nature.exceptions import QiskitNatureError
+
+from qiskit_nature.second_q.operators import BosonicOp
+
+from .qubit_mapper import ListOrDictType, QubitMapper
+
+
+class BosonicMapper(QubitMapper):
+    """
+    Mapper of Bosonic Operator to Qubit Operator
+
+    Workflow:
+        First, the user initializes an instance of BosonicLinearMapper, which call the __init__ method
+        defined in this class.
+        Then, the user calls BosonicLinearMapper.map(). Since map is not defined in BosonicLinearMapper,
+        the call is actually to the map method in this class.
+        QubitMapper.map() calls QubitMapper.map() thank to the super() class.
+        Inside QubitMapper.map() there is a call to _map_single. This method is present in QubitMapper
+        class, but it is overloaded in BosonicLinearMapper.
+        Thus, the call is to BosonicLinearMapper._map_single()
```

Comment:
While I see that this explanation can be useful, I think that the workflow can be inferred from the code, and it does not look consistent with the rest of our documentation, so I would opt to remove it in this case. I am adding though the `attributes` field that Max mentions below (we can discuss it later):

```suggestion
     The following attributes can be read and updated once the ``BosonicMapper`` object
     has been constructed.

        Attributes:
            truncation (int): INSERT DEFINITION HERE. Mention that the min. value is 1.
```